packages/forms-web-app/src/controllers/selected-appeal/appeal-details/index.js

```diff
@@ -0,0 +1,59 @@
+const { LPA_USER_ROLE } = require('@pins/common/src/constants');
+const { formatHeadlineData } = require('@pins/common');
+
+const { VIEW } = require('../../../lib/views');
+const { apiClient } = require('../../../lib/appeals-api-client');
+const { determineUser } = require('../../../lib/determine-user');
+
+/**
+ * @type {import('express').Handler}
+ */
+exports.get = async (req, res) => {
```

Comment:
Have I seen this code before 🤔 

![image](https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/35689158/50b96dbc-14a8-4f4a-bb4d-ca0e64c79c2b)